### ISSUE TITLE
Can't use :json encoder with 0.7.18

### ISSUE THREAD

**Issue by codahale** [pyr/unilog#15]
Looks like the Logstash encoder isn't compatible with the new version of Logback:

```
user=> (require '[unilog.config :as unilog])
nil
user=> (unilog/start-logging! {:console {:encoder :json}})

AbstractMethodError   ch.qos.logback.core.OutputStreamAppender.encoderInit (OutputStreamAppender.java:180)
user=> *e
#error {
 :cause nil
 :via
 [{:type java.lang.AbstractMethodError
   :message nil
   :at [ch.qos.logback.core.OutputStreamAppender encoderInit "OutputStreamAppender.java" 180]}]
 :trace
 [[ch.qos.logback.core.OutputStreamAppender encoderInit "OutputStreamAppender.java" 180]
  [ch.qos.logback.core.OutputStreamAppender setOutputStream "OutputStreamAppender.java" 171]
  [ch.qos.logback.core.ConsoleAppender start "ConsoleAppender.java" 81]
  [unilog.config$eval1734$fn__1735 invoke "config.clj" 297]
  [clojure.lang.MultiFn invoke "MultiFn.java" 233]
  [unilog.config$start_logging_BANG_ invokeStatic "config.clj" 361]
  [unilog.config$start_logging_BANG_ invoke "config.clj" 300]
  [user$eval1766 invokeStatic "form-init7693490714008086449.clj" 1]
  [user$eval1766 invoke "form-init7693490714008086449.clj" 1]
  [clojure.lang.Compiler eval "Compiler.java" 6927]
  [clojure.lang.Compiler eval "Compiler.java" 6890]
  [clojure.core$eval invokeStatic "core.clj" 3105]
  [clojure.core$eval invoke "core.clj" 3101]
  [clojure.main$repl$read_eval_print__7408$fn__7411 invoke "main.clj" 240]
  [clojure.main$repl$read_eval_print__7408 invoke "main.clj" 240]
  [clojure.main$repl$fn__7417 invoke "main.clj" 258]
  [clojure.main$repl invokeStatic "main.clj" 258]
  [clojure.main$repl doInvoke "main.clj" 174]
  [clojure.lang.RestFn invoke "RestFn.java" 1523]
  [clojure.tools.nrepl.middleware.interruptible_eval$evaluate$fn__947 invoke "interruptible_eval.clj" 87]
  [clojure.lang.AFn applyToHelper "AFn.java" 152]
  [clojure.lang.AFn applyTo "AFn.java" 144]
  [clojure.core$apply invokeStatic "core.clj" 646]
  [clojure.core$with_bindings_STAR_ invokeStatic "core.clj" 1881]
  [clojure.core$with_bindings_STAR_ doInvoke "core.clj" 1881]
  [clojure.lang.RestFn invoke "RestFn.java" 425]
  [clojure.tools.nrepl.middleware.interruptible_eval$evaluate invokeStatic "interruptible_eval.clj" 85]
  [clojure.tools.nrepl.middleware.interruptible_eval$evaluate invoke "interruptible_eval.clj" 55]
  [clojure.tools.nrepl.middleware.interruptible_eval$interruptible_eval$fn__992$fn__995 invoke "interruptible_eval.clj" 222]
  [clojure.tools.nrepl.middleware.interruptible_eval$run_next$fn__987 invoke "interruptible_eval.clj" 190]
  [clojure.lang.AFn run "AFn.java" 22]
  [java.util.concurrent.ThreadPoolExecutor runWorker "ThreadPoolExecutor.java" 1142]
  [java.util.concurrent.ThreadPoolExecutor$Worker run "ThreadPoolExecutor.java" 617]
  [java.lang.Thread run "Thread.java" 745]]}
```

**Comment by codahale** [comment]
Upstream issue filed here: logstash/logstash-logback-encoder#205.

**Comment by pyr** [comment]
Now fixed in 0.7.19, thanks for catching this early.
